services/order.py

```diff
@@ -0,0 +1,28 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
```

Comment:
Avoid using mutable default arguments like `list`. Use `None` as the default value and initialize the list inside the function if necessary.